ocfweb/docs/docs/staff/backend/prometheus.md

```diff
@@ -0,0 +1,44 @@
+[[!meta title="Prometheus"]]
+
+We use Prometheus to provide real-time monitoring of our hardware. The master is [[dementors|doc staff/backend/servers]] which
+uses the Node Exporter to collect data from other servers.
+
+We monitor servers, desktops, and staff VMs, but not the hozer boxes.
+Additionally, we don't receive email alerts for staff VMs. Monitoring for the networking switch, blackhole, is currently under development.
+
+## Alerts
+
+Alerts can be viewed at [prometheus.ocf.berkeley.edu/alerts](https://prometheus.ocf.berkeley.edu/alerts). They are configured at [this folder][prometheus-puppet] in the Puppet configs.
```

Comment:
You should also consider linking to prometheus.ocf.berkeley.edu/alertmanager

Reply:
> You should also consider linking to prometheus.ocf.berkeley.edu/alertmanager

What exactly does alertmanager do?

Reply:
https://prometheus.io/docs/alerting/alertmanager/

Alertmanager basically handles making Prometheus alerts useful (notifying us via email, slack, etc.). It can also do other cool things like inhibition (mute other alerts when one exists e.g. mute alerts related to failed file syncing if the host is down).